app/Importers/WheeMoveDataImporter.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Importers;
+
+use GuzzleHttp\Exception\GuzzleException;
+use Symfony\Component\DomCrawler\Crawler;
+
+class WheeMoveDataImporter extends DataImporter
+{
+    protected Crawler $sections;
+
+    public function extract(): static
+    {
+        try {
+            $response = $this->client->get("https://www.wheemove.com/");
+            $html = $response->getBody()->getContents();
+        } catch (GuzzleException) {
+            $this->createImportInfoDetails("400", self::getProviderName());
+            $this->stopExecution = true;
+        }
+
+        $crawler = new Crawler($html);
+
+        $this->sections = $crawler->filter('section[data-id="4b8f82b"], section[data-id="59def84"]');
+
+        if (count($this->sections) === 0) {
+            $this->createImportInfoDetails("204", self::getProviderName());
+
+            $this->stopExecution = true;
+        }
+
+        return $this;
+    }
+
+    public function transform(): void
+    {
+        $existingCityProviders = [];
+        $this->extract();
```

Comment:
Why calling `extract()` second time?

Reply:
Sorry, it was used temporary for testing. I'll remove it